plugin/api.go

```diff
@@ -193,6 +193,9 @@ type API interface {
 	// ReadFileAtPath reads the file from the backend for a specific path
 	ReadFile(path string) ([]byte, *model.AppError)
 
+	// UploadFile will upload a file to a channel using a multipart request, to be later attached to a post.
```

Comment:
Can you document the minimum version required here?

```suggestion
	// UploadFile will upload a file to a channel using a multipart request, to be later attached to a post.
	//
	// Minimum server version: 5.6
```

Reply:
Done as you suggestend @hanzei.

Is the feature planned for 5.6? As general question: When should I document the minimum server version of a api function? No other function got this description.

Reply:
As 5.6 is the next feature release I strongly assume this PR will be includes in it. But I'm just a contributor and don't know for sure.

https://github.com/mattermost/mattermost-server/pull/9616 intruded the documentation of the server version. Since I was the contributor who submitted the PR I'm currently taking care its applied consistently.  Every new API method should have a documentation. I hope I didn't miss one :smile:.

Reply:
Okay :)